pages/congratulations-page.ts

```diff
@@ -0,0 +1,25 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class CongratulationsPage {
+    readonly page: Page;
+    readonly heading: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.heading = page.getByRole("heading", { name: "Congratulations!" });
+    }
+
+    async is_finished() {
```

Comment:
this is read by a developer at first view like, name-of-page.method(), so there is a problem here, it can be read like "congratulation page is finished?" that is probably wrong as it is the installation what is expected to be finished.
The problem is clear, this page should be call by the title like the other pages, InstallationFinished (there might be other pages starting with this name if there is error, don't know exactly how it would look, but using the title is the main thing). Then the name of the method fits better.

Now there is other problem when you call it `is_finished` is it expected a boolean and that is not the case, so it should be called differently, like `expectCongratulations()`, anyway is also an assertion, but with a loop and some error handling, but still an assertion.

Additionally we should be consistent with single quotes or double quotes, and single seems to be the one chosen in Playwright documentation, please follow that for simplicity.